addons/binding/org.openhab.binding.tankerkoenig/ESH-INF/thing/tankerkoenig.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tankerkoenig"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+        <!-- Sample Thing Type -->
+        <thing-type id="tankstelle">
```

Comment:
I agree. The only problem/question is will this change rise problems for users who do an update. Do they have to do a complete new setup because the thing variable name changed?